cli/cmd/lql_create.go

```diff
@@ -30,9 +28,71 @@ var (
 	queryCreateCmd = &cobra.Command{
 		Use:   "create",
 		Short: "create a query",
-		Long:  `Create a query.`,
-		Args:  cobra.NoArgs,
-		RunE:  createQuery,
+		Long: `Create a query.
+
+There are multiple ways you can create a query:
+
+  * Typing the query into your default editor (via $EDITOR)
+  * From a local file on disk using the flag '--file'
+  * From a URL using the flag '--url'
+
+To launch your default editor and create a new query.
+
+    lacework lql create
+
+The following example comes from Lacework's implementation of a policy query:
+
+---
+evaluatorId: Cloudtrail
+queryId: LW_Global_AWS_CTA_AccessKeyDeleted
+queryText: |-
+  LW_Global_AWS_CTA_AccessKeyDeleted {
+      source {
+          CloudTrailRawEvents
+      }
+      filter {
+          EVENT_SOURCE = 'iam.amazonaws.com'
+          and EVENT_NAME = 'DeleteAccessKey'
+          and ERROR_CODE is null
+      }
+      return distinct {
+          INSERT_ID,
+          INSERT_TIME,
+          EVENT_TIME,
+          EVENT
+      }
+  }
+
+This query specifies a dataset named 'LW_Global_AWS_CTA_AccessKeyDeleted'. Policy
```

Comment:
`LW_Global_AWS_CTA_AccessKeyDeleted` is the queryId.  The dataset name is `CloudTrailRawEvents`...

Reply:
support.lacework.com is wrong then 👎🏽 - I will fix the CLI help for now